database/cdb.ksy

```diff
@@ -0,0 +1,54 @@
+meta:
+  id: cdb
+  file-extension: cdb
+  endian: le
+  encoding: ASCII
+  license: CC0-1.0
+doc-ref: https://cr.yp.to/cdb/cdb.txt
```

Comment:
it'd be nice if you added `doc`

Reply:
done

Reply:
I'm not sure that djb cannot sue you (or send a DMCA takedown against this repo) for the strange fact that some parts of this spec match literaly the text from his website.

Reply:
Of course the documentation parts match literally the original specification. I'm just quoting the original words with the mandatory reference to the original source. The problem would be to use the original documentation without citing the source.

Reply:
I'm ok with that

Reply:
https://twitter.com/drescalada/status/1065185136327700480